tailwind.config.js

```diff
@@ -3,7 +3,36 @@ module.exports = {
     './pages/**/*.{js,ts,jsx,tsx}',
     './components/**/*.{js,ts,jsx,tsx}'
   ],
-  theme: {},
+  theme: {
```

Comment:
Add comment with link to style guide: https://design-style-guide.freecodecamp.org/